### ISSUE TITLE
Duplicated html for nested forms

### ISSUE THREAD

**Issue by TheFlow0360** [activeadmin/activeadmin#4856]
I created this code in a partial:

    <%= semantic_form_for [@agent], builder: ActiveAdmin::FormBuilder do |f| %>
        <%= f.semantic_errors %>
        <%= f.inputs 'General Information' do %>
            <%= f.input :name %>
            <%= f.input :description %>
        <% end %>
        <%= f.inputs 'Capture Columns' do %>
            <%= f.has_many :capture_columns, new_record: 'Add Column' do |column| %>
                <%= column.input :column_name %>
                <%= column.input :column_datatype %>
            <% end %>
        <% end %>
        <%= f.actions do %>
            <%= f.action :submit %>
            <li class="cancel"><%= link_to 'Cancel', :back %></li>
        <% end %>
    <% end %>

Basically, this is working, but it looks like this:

[![Forms with duplicated html][1]][1]

The inner HTML for the nested form is duplicated, too:
[![Nested Form with duplicated html][2]][2]

Why is the html duplicated (I checked it, it's exactly the same)? What am I doing wrong?

  [1]: https://i.stack.imgur.com/aKNpD.png
  [2]: https://i.stack.imgur.com/jKbs9.png

**Comment by varyonic** [comment]
Responding on [StackOverflow](http://stackoverflow.com/questions/42765175/activeadmin-nested-form-duplicate) for now.

**Comment by Fivell** [comment]
@TheFlow0360 ,  this can be bug.  looks like has_many with erb is not covered with tests

**Comment by deivid-rodriguez** [comment]
I answered in stackoverflow as well!

**Comment by TheFlow0360** [comment]
If anyone is interested, the problem was that I shouldn't print the inputs for the `has_many` - the code has to look like this:

    <%= f.inputs 'Capture Columns' do %>
        <% f.has_many :capture_columns, new_record: 'Add Column' do |column| %>
            <% column.input :column_name %>
            <% column.input :column_datatype %>
        <% end %>
    <% end %>

**Comment by Fivell** [comment]
@TheFlow0360 , contribution to documentation is appreciated

**Comment by TheFlow0360** [comment]
@Fivell the GitHub wiki or the official page? I don't know how to contribute to the latter.
